health/server.go

```diff
@@ -102,16 +102,19 @@ func (s *Server) Watch(in *healthpb.HealthCheckRequest, stream healthpb.Health_W
 // or insert a new service entry into the statusMap.
 func (s *Server) SetServingStatus(service string, servingStatus healthpb.HealthCheckResponse_ServingStatus) {
 	s.mu.Lock()
-	s.statusMap[service] = servingStatus
```

Comment:
Go style thing:

Prefer:
```go
if x {
   return
}
blah
```
Over:
```go
if !x {
  blah
}
<end function>
```

This makes it more clear that what happened was a nop and nothing else in the function needs to happen.  It also lets you outdent your code which looks nicer.

You can `defer s.mu.Unlock()` to make this style a little simpler.